docs/en/table_design/indexes/indexes_overview.md

```diff
@@ -0,0 +1,150 @@
+  - If multiple sort key columns have similar frequencies as query filter conditions, you can measure the cardinality of these columns.
+  - If the cardinality of the column is high, it can filter more data during the query. If the cardinality is too low, such as for Boolean-type columns, its filtering effect is not ideal. 
+    :::tip
+    
+    However, considering the query characteristics in actual business scenarios, usually, slightly lower cardinality columns are more frequently used as query conditions than high cardinality columns. This is because queries whose filtering is frequently based on high cardinality columns, or even in some extreme scenarios, based on columns with UNIQUE constraints are more like point queries in OLTP databases rather than complex analytical queries in OLAP databases.
+    
+    :::
+  - Also, consider storage compression factors. If the difference in query performance between the order of a low-cardinality column and a high-cardinality one is not readily apparent, placing the low-cardinality column before the high-cardinality one will result in a much higher storage compression rate for the sorted low cardinality column. Therefore, it is recommended to place the low cardinality column in front.
+
+##### Considerations for defining sort key columns at table creation
+
+When defining sort key columns for a Duplicate Key table, an Aggregate table, a Unique Key table, pay attention to the following points:
+
+- The sort key columns must be defined before other columns at table creation.
+- The sort key columns must be the first one or more columns in the table, and the order of the sort key columns must match the sequence of these columns in the table.
+- The data types of the sort key columns do not can be numeric types (excluding DOUBLE and FLOAT), strings, and date types.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/8490d97f-6210-4c9b-ad32-ff4922502697)
do not can -> can